criu/cr-restore.c

```diff
@@ -3133,6 +3133,9 @@ rst_prep_creds_args(CredsEntry *ce, unsigned long *prev_pos)
 {
 	unsigned long this_pos;
 	struct thread_creds_args *args;
+	int n, len;
+	bool groups_are_ok;
+	unsigned int* gids;
```

Comment:
Shouldn't we  use `gid_t` instead of `unsigned int`

Reply:
Ah, I think that's right. We memcmp gids to ce->groups. Both `gid_t` and ce->groups are `uint32_t`.